_guidelines-v4/12-facsimilesrecordings/01-facsimiles/01-facsimileelements.md

```diff
@@ -36,6 +36,11 @@ Because the coordinate space of a zone is defined relative to that of a surface,
 
 {% include mei example="facsimiles/facsimiles-sample249.xml" valid="" %}
 
+Additionally, the area represented by a {% include link elem="zone" %} element may not be normally oriented to its parent surface. The amount by which the area has been rotated clockwise in degrees is expressed by the **@rotate** attribute. By default this amount is 0&deg;.
```

Comment:
The amount by which the contents of this area…

Reply:
should be equal to the attribute description in the proposed schema PR

Reply:
I just added a commit trying to do this. Is it okay?